Assets/Game/Scripts/Timer.cs

```diff
@@ -0,0 +1,52 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using UnityEngine.SceneManagement;
+
+public class Timer : MonoBehaviour
+{
+    private float remainingTime;
+    private bool timerIsRunning = false;
+    public Text OSTimer;
```

Comment:
please use Text Mesh Pro for the UI and wire this class with the other elements. If you open the GameManagers prefab you can see there is the HUD that can be used to show the text

Reply:
Working on wiring it in, will probably take a bit of time